src/main/java/core/basesyntax/service/implementation/CreateReportServiceImplementation.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CreateReportServiceImplementation implements CreateReportService {

```

Comment:
do not use full word "Implementation", fix everywhere
```suggestion
public class CreateReportServiceImpl implements CreateReportService {
```